scipy/signal/tests/test_signaltools.py

```diff
@@ -533,14 +923,31 @@ def test_invalid_shapes(self):
         assert_raises(ValueError, fftconvolve, *(a, b), **{'mode': 'valid'})
         assert_raises(ValueError, fftconvolve, *(b, a), **{'mode': 'valid'})
 
+    def test_invalid_shapes_axes(self):
+        # By "invalid," we mean that the axes
+        # over which the convolution is NOT
+        # applied cannot be broadcast together.
+        a = np.zeros([5, 6, 2, 1])
+        b = np.zeros([5, 6, 3, 1])
+        assert_raises(ValueError, fftconvolve, a, b, axes=[0, 1])
+        assert_raises(ValueError, fftconvolve, b, a, axes=[0, 1])
+
     def test_mismatched_dims(self):
         assert_raises(ValueError, fftconvolve, [1], 2)
         assert_raises(ValueError, fftconvolve, 1, [2])
         assert_raises(ValueError, fftconvolve, [1], [[2]])
         assert_raises(ValueError, fftconvolve, [3], 2)
+        assert_raises(ValueError, fftconvolve,
```

Comment:
Same with these new ones, let's use `pytest.raises` context manager with `match` arg, it makes it obvious to the next coder (or when it fails) what it's meant to do

Reply:
Done.